planning/mission_planner/src/lanelet2_plugins/utility_functions.cpp

```diff
@@ -86,3 +86,30 @@ lanelet::ConstLanelet combine_lanelets(const lanelet::ConstLanelets & lanelets)
   auto combined_lanelet = lanelet::Lanelet(lanelet::InvalId, left_bound, right_bound);
   return std::move(combined_lanelet);
 }
+
+std::vector<geometry_msgs::msg::Point> convertCenterlineToPoints(const lanelet::Lanelet & lanelet)
+{
+  std::vector<geometry_msgs::msg::Point> centerline_points;
+  geometry_msgs::msg::Point center_point;
+  for (const auto & point : lanelet.centerline()) {
+    center_point.x = point.basicPoint().x();
+    center_point.y = point.basicPoint().y();
+    center_point.z = point.basicPoint().z();
+    centerline_points.push_back(tier4_autoware_utils::getPoint(center_point));
+  }
+  return centerline_points;
+}
+
+geometry_msgs::msg::Pose convertBasicPoint3dToPose(
+  const lanelet::BasicPoint3d & point, const double & lane_yaw)
+{
+  // calculate new orientation of goal
+  geometry_msgs::msg::Pose pose;
+  pose.position.x = point.x();
+  pose.position.y = point.y();
+  pose.position.z = point.z();
+
+  pose.orientation = tier4_autoware_utils::createQuaternionFromRPY(0.0, 0.0, lane_yaw);
```

Comment:
createQauterninoFromYaw can be used.

Reply:
@takayuki5168 thanks for the review! I updated it.